sbmtm.py

```diff
@@ -523,13 +524,35 @@ def group_to_group_mixture(self,l=0,norm=True):
         else:
             return n_td_tw
 
-    def pmi_td_tw(self,l=0):
-        '''
-        Point-wise mutual information between topic-groups and doc-groups, S(td,tw)
-        This is an array of shape Bd x Bw.
-
-        It corresponds to
-        S(td,tw) = log P(tw | td) / \tilde{P}(tw | td) .
+    def plot_topic_dist(self, l):
```

Comment:
The new function plot_topic_dist seems to partially remove the function pmi_td_tw

Reply:
Thanks for the addition. 
Perhaps we can add it without altering the existing functions?

Reply:
should be fixed now